.goreleaser.yml

```diff
@@ -0,0 +1,56 @@
+builds:
+  - main: ./server/
+    binary: server/wysteria-server
+    ldflags: -s -w -X main.Version={{.Version}}
+    goos:
+      - windows
+      - darwin
+      - linux
+    ignore:
+      - goos: darwin
+        goarch: 386
+    env:
+      - CGO_ENABLED=0
+
+archive:
+  wrap_in_directory: true
+  replacements:
+    386: 32-bit
+    amd64: 64-bit
+    darwin: osx
+  format_overrides:
+    - goos: windows
+      format: zip
+  files:
+    - LICENSE
+    - README.md
+    - server/wysteria-server.ini.example
+    - docker/*
+
+release:
+  github:
+    owner: voidshard
+    name: wysteria
+  draft: true
```

Comment:
Leaving this set to true means that your release in github will be in the draft state. You then have a chance to review/edit the release notes before actually publishing it.